sae_lens/training/optim.py

```diff
@@ -157,3 +157,61 @@ def load_state_dict(self, state_dict: dict[str, Any]):
         """Loads all state apart from attached SAE."""
         for k in state_dict:
             setattr(self, k, state_dict[k])
+
+
+class L0Scheduler:
```

Comment:
This feels very heavy-weight to add another class entirely that's basically identical to `L1Scheduler`. We should just make the `L1Scheduler` into `CoefficientScheduler` or something so it can be used in multiple places